__tests__/app.test.js

```diff
@@ -96,3 +84,28 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET 200: responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.articles).toBeSortedBy("created_at", {
```

Comment:
This is my opinion so not a must, but personally I would want to separate the testing for a default sort order out into a separate 200 test as it's a different behaviour, as if I did something later on with this endpoint such as adding functionality for queries later on I would also want to test them separately